x-pack/plugins/apm/e2e/cypress/integration/rum_dashboard.feature

```diff
@@ -5,3 +5,19 @@ Feature: RUM Dashboard
     When the user inspects the real user monitoring tab
     Then should redirect to rum dashboard
       And should have correct client metrics
+
+  Scenario: Page load distribution
+    Given a user browses the APM UI application for RUM Data
+    When the user inspects the real user monitoring tab
+    Then should redirect to rum dashboard
+      And should display percentile for page load chart
+      And should display tooltip on hover
+      And should display chart legend
+    Given a user click page load breakdown filter
+    When the user selected the breakdown
+    Then breakdown series should appear in chart
+
+    Given a user click the filter
```

Comment:
It's a good idea to add a Scenario before this Given, so we separate it from other scenarios.